### ISSUE TITLE
Adding "advancedPHP" to your Theme

### ISSUE THREAD

**Issue by dennisosaj** [dracula/nova#5]
Heyho,
I saw your theme and noticed a growing popularity.

I'm asking if you could add "advancedPHP" to your theme. It's one of my own extensions and I want to provide more theme coverage for my users.

You can find a theme development documentation in its repo: https://github.com/dennisosaj/advancedphp.novaextension 

I would really appreciate it. Thanks!

**Comment by fefed22** [comment]
Hi, 
it will be great and helpful !!
Best regards

**Comment by bdielman** [comment]
@dennisosaj  - I'll look into this! Thanks!

**Comment by fefed22** [comment]
Hello,

@bdielman  Any news ?
are you going to make it for Nova 9 or are you waiting for Nova 10 ?

Best regards

**Comment by bdielman** [comment]
Yep. Waiting to see what Nova 10 has in store with the migration to treesitter. I have a feeling a lot of language/theme extensions will need to be updated post-release.

**Comment by fefed22** [comment]
Hello,
i'm coming for news ...
:)

**Comment by bdielman** [comment]
No news just yet. Very busy with other projects at the moment, but when I get time to properly test it out I'll update this thread.

**Comment by bdielman** [comment]
@fefed22 - Dusting off the issues in this repo. Sorry for not keeping up with things. Looks like the Advanced PHP plugin hasn't been updated recently (probably the last time I updated this repo 😬). Are you still looking for support for this extension?
